spec/services/account_creation/device_profiling_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe AccountCreation::DeviceProfiling do
+  let(:threatmetrix_session_id) { '13232' }
+  let(:threatmetrix_proofer_result) do
+    instance_double(Proofing::DdpResult, success?: true, transaction_id: 'ddp-123')
+  end
+  let(:threatmetrix_proofer) do
+    instance_double(
+      Proofing::LexisNexis::Ddp::Proofer,
+      proof: threatmetrix_proofer_result,
+    )
+  end
+
+  subject(:device_profiling) { described_class.new }
+
+  describe '#proof' do
+    subject(:proof) do
+    end
+
```

Comment:
Doesn't look like we're using this.

We could get rid of the `@device_profiling_result` below and include it here instead? Since `subject` is memoized and lazily evaluated, wouldn't need to worry about it happening before stubs or being called multiple times.

```suggestion
    subject(:result) do
      device_profiling.proof(
        request_ip: Faker::Internet.ip_v4_address,
        threatmetrix_session_id: threatmetrix_session_id,
        user_email: Faker::Internet.email,
      )
    end

```